templates/home/api-en.html

```diff
@@ -0,0 +1,217 @@
+---
+{
+	"title": "Home page - Meta information",
+	"language": "en",
+	"altLangPrefix": "fr",
+	"altLangPage": "api-fr.html",
+	"breadcrumb": [
+		{ "title": "GCWeb home", "link": "../index-en.html" }
+	],
+	"secondlevel": false,
+	"dateModified": "2022-04-27",
+	"share": "true"
+}
+---
+
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable for GCWeb V5</dd>
+
+	<dt>Type</dt>
+	<dd>Template <!-- Common design patter --></dd>
+
+	<dt>Last review</dt>
+	<dd>2022-04-27</dd>
+
+	<dt>Conforming to</dt>
+	<dd>Going to be: Content and IA spec 2.0</dd>
+
+	<dt>Specification</dt>
+	<dd><a href="https://design.canada.ca/mandatory-templates/home-page.html">Home page for Canada.ca</a>
+</dl>
+
+<h2>Technical specification</h2>
+
+<h3>Component list and latest version</h3>
+<ul>
+	<li><a href="#home-page-banner">Home page banner</a> - Version 2.0</li>
+	<li><a href="#promotional-banner">Promotional banner</a> - Version 2.0</li>
+	<li>Most requested - Version 1.0</li>
+	<li>Service and information - Version 1.0</li>
+	<li>Focus on - Version 1.0</li>
+	<li><a href="#your-government">Your government</a> - Version 2.0</li>
+	<li>Government initiatives - Version 1.0</li>
+</ul>
+
+<h3 id="home-page-banner">Home page banner</h3>
+<dl>
+	<dt>Version</dt>
+	<dd>2</dd>
+	<dt>Updated</dt>
+	<dd>2019-05-09</dd>
+</dl>
+<p>The background image is loaded with the provisional plugin bgimg which set the background image. The background image is hosted in the WMS at this location. <code>https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg</code>.</p>
+<p>The intend is to publish background image that represent natural Canadian landscape and may get updated montly or seasonal.</p>
+<h4>Previous version</h4>
+<details>
+	<summary>Version 1 - 2018-12-27</summary>
+	<p>The default background is gray (#eee) without image. The background image is hosted in the WMS at this location. <code>https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg</code>.</p>
+	<p>The intend is to publish background image that represent natural Canadian landscape and may get updated montly or seasonal.</p>
+</details>
+
+<h3 id="promotional-banner">Promotional banner</h3>
+<p>The promotional banner are over, at the bottom, of the home page banner background image.</p>
+<dl>
+	<dt>Version</dt>
+	<dd>2</dd>
+	<dt>Updated</dt>
+	<dd>2019-05-09</dd>
+	<dt>Notes</dt>
+	<dd>Provisional color was removed.</dd>
+</dl>
+<pre><code>
+&lt;div class="provisional bg-cover" data-bgimg="https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg"&gt;
```

Comment:
```suggestion
<pre><code>&lt;div class="provisional bg-cover" data-bgimg="https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg"&gt;
```

Optional: I do think we can remove the need to use the "provisional" CSS because those plugin has been stabilized. But if we do, we will need to update the this component by one minor version
```suggestion
<pre><code>&lt;div class="bg-cover" data-bgimg="https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg"&gt;
```

Reply:
@duboisp Let's consider this one out-of-scope for this PR, and let's have a subsequent discussion about it!

cc: @polmih

Reply:
@GormFrank that is fine. 
Note for ourself: we need to update our Public API to exclude that type of template change which impact only one page or impact a small set of pages based on Canada.ca statistic usage.